src/test/java/edu/ucla/library/services/metadata/MetadataSetterTest.java

```diff
@@ -163,12 +163,14 @@ public void testPopulateMediaFormat() throws Exception {
      */
     @Test
     public void testGetMetaFromBadFileMultipleMediaPaths() throws Exception {
-        final String errorMessage = "Problem reading media file";
+        final String badMedia = "bad-mp3-file.mp3";
+        final String probeError = "/usr/bin/ffprobe returned non-zero exit status. Check stdout.";
```

Comment:
You can still put this in the messages file and have a LOGGER.getMessage() within the LOGGER.getMessage(), fwiw.

Reply:
I wouldn't do that for line 166. Just for 167.